src/compose/service.coffee

```diff
@@ -28,17 +28,23 @@ createRestartPolicy = (name) ->
 		name = 'always'
 	return { Name: name, MaximumRetryCount: 0 }
 
+ensureCommandIsArray = (s) ->
+	if _.isString(s)
+		s = _.map parseCommand(s.replace(/(\$)/g, '\\$1')), (arg) ->
```

Comment:
I think this would be easier if you moved this out to a utility function like:

Reply:
*suspense* :P